apps/condo/domains/ticket/access/IncidentClassifierIncident.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * Generated by `createschema ticket.IncidentClassifierIncident 'incident:Relationship:Incident:CASCADE; classifier:Relationship:IncidentClassifier:PROTECT;'`
+ */
+const { get } = require('lodash')
+
+const { isSoftDelete } = require('@open-condo/keystone/access')
+const { throwAuthenticationError } = require('@open-condo/keystone/apolloErrorFormatter')
+const { getById } = require('@open-condo/keystone/schema')
+
+const {
+    queryOrganizationEmployeeFor,
+    queryOrganizationEmployeeFromRelatedOrganizationFor,
+    checkPermissionInUserOrganizationOrRelatedOrganization,
+} = require('@condo/domains/organization/utils/accessSchema')
+
+
+async function canReadIncidentClassifierIncidents ({ authentication: { item: user } }) {
+    if (!user) return throwAuthenticationError()
+    if (user.deletedAt) return false
+    if (user.isAdmin || user.isSupport) return {}
+
+    return {
+        incident: {
```

Comment:
Maybe add an `organization` field to `IncidentClassifierIncident` and avoid extra join?
From `incident` field by using `addOrganizationFieldPlugin`